InternxtDesktop/AppDelegate.swift

```diff
@@ -57,7 +57,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         ErrorUtils.start()
 
         checkVolumeAndEjectIfNeeded()
-        backupsService.assignUrls()
+        initializeBackups()
```

Comment:
we will need a better way to organize the AppDelegate, is starting to grow a lot, nothing to change here, but we should start thinking a solution. This is ok tho